website/events-listener/src/logger.js

```diff
@@ -0,0 +1,29 @@
+import winston from 'winston';
```

Comment:
I've heard [pino](https://github.com/pinojs/pino) much faster than winston

Reply:
I'll take a look at this